gitea/repo_commit.go

```diff
@@ -17,9 +24,8 @@ type CommitMeta struct {
 
 // CommitUser contains information of a user in the context of a commit.
 type CommitUser struct {
-	Name  string `json:"name"`
-	Email string `json:"email"`
-	Date  string `json:"date"`
+	*Identity
```

Comment:
Why not just use `Identity` but `*Identity` Since it should not be `nil`?

Reply:
Good point and now I understand this better. Fixed.